docs/requirements-docs.txt

```diff
@@ -1,8 +1,8 @@
 #
-# This file is autogenerated by pip-compile with Python 3.9
+# This file is autogenerated by pip-compile with Python 3.8
 # by the following command:
 #
-#    pip-compile requirements-docs.in
+#    pip-compile --constraint=requirements.txt --output-file=docs/requirements-docs.txt docs/requirements-docs.in
```

Comment:
why is this needed?  This reqs file is only for generating the readthedocs site...(ref seed-farmer/.readthedocs.yaml)